lib/installation_utils.pm

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+package installation_utils;
```

Comment:
IMHO defining a package for a single function is over-engineering. Unless we know of other functions that should also be moved here I suggest to just define that in an existing package or lib-file

Reply:
This function is only for remote installation, later the function **is_remote_backend** will come.

Where should I put those functions better?